apps/3d_rec_framework/include/pcl/apps/3d_rec_framework/pc_source/mesh_source.h

```diff
@@ -231,7 +231,7 @@ class MeshSource : public Source<PointInT> {
     std::vector<std::string> files;
     std::string start;
     std::string ext = std::string("ply");
-    bf::path dir = path_;
+    pcl_fs::path dir = path_;
```

Comment:
```suggestion
    bf::path dir = path_;
```
The function from apps/3d_rec_framework/include/pcl/apps/3d_rec_framework/pc_source/source.h  is used. With `pcl_fs` there is an error: `cannot convert 'std::filesystem::path' to 'boost::filesystem::path&'`